### PR TITLE
Rephrase RSC-005/016/017

### PR DESCRIPTION
**Pull request by tofi86**
because the previous phrasing was a bit misleading and had redundant apostrophes

```
RSC_005=Error while parsing file '%1$s'.
```
Example:
```
Error while parsing file 'element "package" incomplete; missing required element "spine"'.
```

The error didn’t appear in file '%1$s' but instead the variable was holding the error **message** which appeared when parsing the file. The filename however was never part of the message.
New phrasing should be clearer now:

```
RSC_005=Error while parsing file: %1$s
```
Example:
```
Error while parsing file: element "package" incomplete; missing required element "spine"
```

### CODE REVIEW

**Comment by tofi86** [comment]
@rdeltour can you do a quick review soon as this blocks #650 rebase?

Otherwise I'm going to merge this as it's only rephrasing and nothing big...

File: src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties [hunk 1]
```diff
@@ -273,7 +273,7 @@ RSC_001=File '%1$s' could not be found.
 RSC_002=Required META-INF/container.xml resource could not be found.
 RSC_003=No rootfile tag with media type 'application/oebps-package+xml' was found in the container.
 RSC_004=File '%1$s' could not be decrypted.
-RSC_005=Error while parsing file '%1$s'.
+RSC_005=Error while parsing file: %1$s
```
Comment (rdeltour) [inline src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties]: Maybe an even shorter: `Parsing error: %1$s`?

File: src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties [hunk 2]
```diff
@@ -286,8 +286,8 @@ RSC_012=Fragment identifier is not defined.
 RSC_013=Fragment identifier is used in a reference to a stylesheet resource.
 RSC_014=Fragment identifier defines an incompatible resource type.
 RSC_015=A fragment identifier is required for svg use tag references.
-RSC_016=Fatal Error while parsing file '%1$s'.
-RSC_017=Warning while parsing file '%1$s'.
+RSC_016=Fatal Error while parsing file: %1$s
```
Comment (rdeltour) [inline src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties]: Similarly, even shorter: `Fatal parsing error: %1$s`?

File: src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties [hunk 3]
```diff
@@ -286,8 +286,8 @@ RSC_012=Fragment identifier is not defined.
...
+RSC_017=Warning while parsing file: %1$s
```
Comment (rdeltour) [inline src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties]: Similarly, even shorter: `Parsing warning: %1$s`?

File: src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties (hunk 1 above)
Comment (tofi86) [inline src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties]: yep, sounds good! Will shorten all the messages!

File: src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties (hunk 3 above)
Comment (tofi86) [inline src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties]: `Parsing warning` or `Parser warning`? What sounds better?

Comment (rdeltour) [inline src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties]: yeah, "parsing warning" might sound odd, I don't know. In any case, we need to be consistent with the previous ones.

Comment (tofi86) [inline src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties]: @mattgarrish two non-native speakers wondering about good sounding and correct error messages: `Warning while parsing file: {message}` or `Parsing warning: {message}` or `Parser warning: {message}`. Your 2 cents?

Comment (mattgarrish) [inline src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties]: Honestly, I don't even know if the average user knows what parsing is. Couldn't you just emit the level followed by the message? That's really all anyone needs to know.

If that's too heretical, "X while parsing file" is the more natural sounding, albeit a bit longer.

Comment (tofi86) [inline src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties]: Then I'd say we stick to `X while parsing file: {message}`. @rdeltour okay for you?

Comment (rdeltour) [inline src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties]: > Then I'd say we stick to X while parsing file: {message}

Sure, go ahead.

**Pull request merged**